### ISSUE TITLE
More than 4 categories?

### ISSUE THREAD

**Issue by m-dr** [KoreTeknology/Blender-GN-Library-Addon#3]
Hey, this is already better than the regular node presets, but it'd be very nice to be able to add more than 4 categories.
Also, it'd be very cool to be able to turn off grouping into categories in the settings - presenting all presets in a flat hierarchy like the regular node presets addon. That way it'd be as flexible as possible for the user, I think.

**Comment by chen135347542093** [comment]
Yes, it would be cool if you could customize categories and add unlimited categories

**Comment by KoreTeknology** [comment]
hi, completely ok with unlimitited categories, i will try my best to update the code when i found the best solution to get some kind of users multi-path block to be added for each type and nodes windows, after all, i need a menu like this for each type of node groups, but separated in the UI

**Comment by chen135347542093** [comment]
Looking forward to it, because I have hundreds of geometric nodes
